go.sum

```diff
@@ -737,6 +738,8 @@ google.golang.org/protobuf v1.23.0/go.mod h1:EGpADcykh3NcUnDUJcl1+ZksZNG86OlYog2
 google.golang.org/protobuf v1.23.1-0.20200526195155-81db48ad09cc/go.mod h1:EGpADcykh3NcUnDUJcl1+ZksZNG86OlYog2l/sGQquU=
 google.golang.org/protobuf v1.24.0 h1:UhZDfRO8JRQru4/+LlLE0BRKGF8L+PICnvYZmx/fEGA=
 google.golang.org/protobuf v1.24.0/go.mod h1:r/3tXBNzIEhYS9I1OUVjXDlt8tc493IdKGjtUeSXeh4=
+google.golang.org/protobuf v1.25.0 h1:Ejskq+SyPohKW+1uil0JJMtmHCgJPJ/qWTxr8qp+R4c=
```

Comment:
why do we have so many version of protobuf here? which one is used when I do `import "google.golang.org/protobuf"`?

(sorry if it's a noob golang question)

Reply:
I have no idea how to parse or read the sum file -- I don't really think you need to look at this ever. However, the `go.mod` tells you exactly the version that is being used.

Reply:
this is because some dependencies are probably using previous versions of protobuf